ppl/src/main/antlr/OpenSearchPPLParser.g4

```diff
@@ -419,17 +414,44 @@ trigonometricFunctionName
     ;
 
 dateAndTimeFunctionBase
-    : ADDDATE | CONVERT_TZ | DATE | DATE_ADD | DATE_FORMAT | DATE_SUB
-    | DATETIME | DAY | DAYNAME | DAYOFMONTH | DAYOFWEEK | DAYOFYEAR | FROM_DAYS | FROM_UNIXTIME
-    | HOUR | MAKEDATE | MAKETIME | MICROSECOND | MINUTE | MONTH | MONTHNAME | PERIOD_ADD
-    | PERIOD_DIFF | QUARTER | SECOND | SUBDATE | SYSDATE | TIME | TIME_TO_SEC
-    | TIMESTAMP | TO_DAYS | UNIX_TIMESTAMP | WEEK | YEAR
-    ;
-
-// Functions which value could be cached in scope of a single query
-constantFunctionName
-    : CURRENT_DATE | CURRENT_TIME | CURRENT_TIMESTAMP | LOCALTIME | LOCALTIMESTAMP | UTC_TIMESTAMP | UTC_DATE | UTC_TIME
-    | CURDATE | CURTIME | NOW
+    : datetimeConstantLiteral
```

Comment:
why define datetimeConstantLiteral? could it be part of dateAndTimeFunctionBase? i did not find code handle datetimeConstantLiteral in AstExpressionBuilder.

Reply:
Good point. Removed in e7ccfc2bd2cc20c2c581210a58b5afda3ccf4d50.